app/services/word_watcher.rb

```diff
@@ -8,16 +8,12 @@ def initialize(raw)
   end
 
   def self.disable_cache
-    @disabled = true
-  end
-
-  def self.enable_cache
-    @disabled = false
+    @cache_disabled = true
```

Comment:
Haven't followed the previous PR, but would it make sense to invert that ivar? i.e. `@cache_enabled = false`?

I have an allergy for double negatives, like `cache_disabled = false`
I try to limit my mental boolean math 😂

Reply:
Using `cache_enabled` complicates the code further because that ivar needs to be initialize to `true`. I'll prefer to keep this as it is.

Reply:
Arguably explicitly initializing ivars is a good thing. After all you get a ruby warnings if you don't. And in the far far future 😅 when we add types to ruby code, we'd want it to be true/false and not true/false/nil 😉